zipline/data/treasuries.py

```diff
@@ -67,9 +67,10 @@ def get_treasury_data(start_date, end_date):
         "&from="  # An unbounded query is ~2x faster than specifying dates.
         "&to="
         "&filetype=csv"
+        "&label=include"
         "&layout=seriescolumn"
         "&type=package",
-        skiprows=1,  # First row is a useless header.
+        skiprows=5,  # First 5 rows are useless headers.
```

Comment:
This is just a problem I ran into and noticed when I wanted to be sure that the benchmarking works (by running a small algo I made locally). When I deleted the cached `treasury_curves.csv` from `~/.zipline`, I ran into the `'Time Period' not in list` error when trying to download new treasury curves data. 

I downloaded a csv of the treasury data manually to check it and it has 5 useless headers at the beginning of the file. Removing those headers fixed the download issue.